perun-wui-registrar/src/main/java/cz/metacentrum/perun/wui/registrar/pages/steps/SummaryStep.java

```diff
@@ -660,10 +660,19 @@ public void onFinished(JavaScriptObject result) {
 
 				@Override
 				public void onError(PerunException error) {
-					errorReporter.setVisible(true);
-					errorReporter.setHTML(ErrorTranslator.getTranslatedMessage(error));
-					errorReporter.setReportInfo(error);
-					resendButton.setProcessing(false);
+					if ("ApplicationNotNewException".equalsIgnoreCase(error.getName())) {
```

Comment:
looks good now, one more thing is the last dialog about "waiting for admin to approve the application", I think that can be removed so it is not confusing: 
![image](https://user-images.githubusercontent.com/73999257/195821511-bf81a786-7e4d-4d76-94a8-1203ea828d42.png)